consent/strategy_default_test.go

```diff
@@ -522,6 +544,10 @@ func runLogout(t *testing.T, method string) {
 
 			assert.EqualValues(t, tc.expectStatusCode, resp.StatusCode, "%s\n%s", resp.Request.URL.String(), out)
 			assert.EqualValues(t, tc.expectBody, strings.Trim(string(out), "\n"), "%s\n%s", resp.Request.URL.String(), out)
+
+			if tc.expectRequestURI != "" {
+				assert.EqualValues(t, tc.expectRequestURI, resp.Request.URL.RequestURI(), "%s\n%s", resp.Request.URL.String(), out)
+			}
```

Comment:
The `if` could be removed if a `expectRequestURI` would be added to all test-cases.